terraform/modules/rds/variables.tf

```diff
@@ -0,0 +1,14 @@
+variable "db_name" {
+  type    = string
+  default = "  "
```

Comment:
If you set a default value, the variable will be considered as optional and we will not have a prompt to manually set it using the console.
Just add a type and a short description to describe what the variable is for.

Reply:
I understood by the criteria that the values ​​should be changed manually through the AWS console, should it be through the terminal instead?

Reply:
Hi @alanlima , I think it's fine if we leave it with the default value. If it has a default value, the variable is considered to be optional and the default value will be used if no value is set when calling the module or running Terraform.

I think the variables should be something like:

```
variable "db_host" {
  type    = string
  description = "This variable represents the DB_HOST"
  default= " "
}
```
In any case, we need to have "value" or "default". 

@mfreitassm, I understood the same thing:

"Value of parameters to be manually changed through the console (terraform leave logging trace of the values)"

Reply:
Yeah, I misunderstood that. @mfreitassm explained that to me.